### PR TITLE
docs: fix typo in starlark variable name

### PR DESCRIPTION
**Pull request by mksmtn**

### CODE REVIEW

File: site/en/extending/config.md [hunk 1]
```diff
@@ -792,8 +792,7 @@ This produces the following [configured targets](/reference/glossary#configured-
 ...
 ```
 
-`//pkg:dep` produces \\(2^n\\) configured targets: `config.owner=`
-"\\(b_0b_1...b_n\\)" for all \\(b_i\\) in \\(\{0,1\}\\).
+`//pkg:dep` produces `2^n` configured targets: `config.owner="(b_0b_1...b_n)"` for all `b_i` in `{0,1}`.
```
Comment (Wyverald) [inline site/en/extending/config.md:795]: This was intended to be math text, but it was not enabled for this page before. Could you revert the changes in this section (other than the link fix for the "implement a flag" part), and add this text to the top (near line 6):

    <devsite-mathjax config="TeX-AMS-MML_SVG"></devsite-mathjax>

Comment (mksmtn) [reply site/en/extending/config.md]: Fixed. Some CI steps failed, but I guess not due to my changes, they seem to be Java related

**Review by Wyverald — APPROVED** [review]

File: site/en/extending/config.md (hunk 1 above)
Comment (Wyverald) [reply site/en/extending/config.md]: yeah, those should be unrelated. Thanks!
